src/components/AlarmSetting.vue

```diff
@@ -1,28 +1,25 @@
 <script setup lang="ts">
 // import { toRaw } from 'vue'
-import transportJson from '../transport.json'
-import SortBar from './SortBar.vue'
-import CaretIcon from '~icons/srun/caret'
-import SearchIcon from '~icons/srun/search'
-import CheckmarkIcon from '~icons/srun/checkmark'
+import transportJson from "../transport.json";
+import SortBar from "./SortBar.vue";
+import CaretIcon from "~icons/srun/caret";
+import SearchIcon from "~icons/srun/search";
+import CheckmarkIcon from "~icons/srun/checkmark";
 // import type { Settings } from '~/composables/alarmModel'
 // import type { Route } from '~/composables/alarmModel'
 // import { Settings, Transport } from '~/composables/alarmModel'
 
-const emits = defineEmits<{
-  (event: 'interaction', ibl: string, ion: string, ...args: any[]): void
-}>()
-
-interface Props {
-  type: string
-  input: any
+export interface Props {
+  type: string;
+  input: any;
 }
-const {
-  type,
-  input,
-} = defineProps<Props>()
+const { type, input } = defineProps<Props>();
 
-const rndVar = $ref(input)
+const emits = defineEmits<{
+  (event: "interaction", ibl: string, ion: string, ...args: any[]): void;
+}>();
+
+const rndVar = $ref(input);
```

Comment:
Experimental feature?

Personally don't know but this is more familiar to me
```suggestion
const rndVar = input.value;
```